app/views/admin/resources/new.html.haml

```diff
@@ -1 +1 @@
-= render 'form'
+= render 'form', resource: @resource
```

Comment:
Why are you doing this? 
And how are we going to access the edit action after this change?